.changeset/moody-rules-suffer.md

```diff
@@ -0,0 +1,6 @@
+---
+'@remirror/extension-mention-atom': patch
+'@remirror/react-hooks': patch
```

Comment:
Let's add `@remirror/react` here too. I feel the number of published packages are usually too large, so I changed our publish strategy some while ago. Now only package that explicitly added here will be published